docs/conf.py

```diff
@@ -0,0 +1,111 @@
+import os
+import sys
+
+# Configuration file for Sphinx documentation builder.
+
+# -- Setup function ----------------------------------------------------------
+
+# Defines custom steps in the process.
+
+
+def autodoc_skip_member(app, what, name, obj, skip, options):
```

Comment:
These functions may or may not be needed, they are boilerplate I use for other projects; I find them useful.